api/client/samples/crop_w_series/top_5_crop_w_series.ipynb

```diff
@@ -0,0 +1,1415 @@
+{
```

Comment:
<p>Is there a reason that in the table, country and unit are shown with their names while the source is shown in its <code>id</code>  as opposed to its name?</p><p>I suppose it's <code>FAO</code>  right? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/gro-intelligence/api-client/pull/312/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>